index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="stylesheets/style.css">
+    <title>Hello microverse</title>
+</head>
+<body>
+    <header>
+        <h2>Hello World!</h2>
```

Comment:
Following the project requirement, it's supposed to use h1 tag instead of h1 tag for the title. `h1` is a good indicator of what the most important text on a page is.